cmake/Finddpdk.cmake

```diff
@@ -84,6 +84,14 @@ endif ()
 list (APPEND dpdk_REQUIRED
   dpdk_INCLUDE_DIR)
 
+# Depending on whether libbsd-dev exists at build time, DPDK may be built with a dependency on
+# libbsd. Note that other libraries in dpdk_PC_LIBRARIES are handled using separate logic
+# (see rte_libs above), thus the additional dependencies must be handled on a case by case basis.
+set (dpdk_dependencies "")
+if ("bsd" IN_LIST dpdk_PC_LIBRARIES)
+  list (APPEND dpdk_dependencies "bsd")
+endif ()
```

Comment:
i'd suggest move this to a place 

- either closer to where it is consumed: right before `if (dpdk_FOUND ...)`
- or closer to where `dpdk_PC_*` variables are initialized: right after `pkg_check_modules (dpdk_PC ...)` 

so that the data dependency are more clear to human readers.

Reply:
Agreed, fixed now.